src/main/java/stsjorbsmod/powers/SpiritShieldPower.java

```diff
@@ -0,0 +1,74 @@
+package stsjorbsmod.powers;
+
+import basemod.interfaces.CloneablePowerInterface;
+import com.badlogic.gdx.graphics.Texture;
+import com.badlogic.gdx.graphics.g2d.TextureAtlas;
+import com.megacrit.cardcrawl.actions.common.ReducePowerAction;
+import com.megacrit.cardcrawl.actions.common.RemoveSpecificPowerAction;
+import com.megacrit.cardcrawl.core.AbstractCreature;
+import com.megacrit.cardcrawl.core.CardCrawlGame;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.localization.PowerStrings;
+import com.megacrit.cardcrawl.powers.AbstractPower;
+import stsjorbsmod.JorbsMod;
+import stsjorbsmod.util.TextureLoader;
+
+import static stsjorbsmod.JorbsMod.makePowerPath;
+
+public class SpiritShieldPower extends AbstractPower implements CloneablePowerInterface {
```

Comment:
Prefer using `CustomJorbsModPower` and `StaticPowerInfo` to reduce boilerplate (I updated all the existing powers to use them yesterday, any of them should be good examples)

Reply:
(this will require renaming the power images to `SpiritShield_32.png` and `SpiritShield_84.png`)